etag.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/sha1"
+	"encoding/binary"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+var randomValue []byte
+
+// run on the server startup
+func init() {
+	if conf.ETagEnabled {
```

Comment:
Are you sure this code will be executed after `conf` initialization?

Reply:
Well, I am afraid to say a banal phrase, but "it works on my computer" :smile:  Seriously, I run that code and I use it for my project for testing, and it works as expected, so the answer is "yes"

Reply:
But surely, I can double check

Reply:
Let's move it to `config.go`, and move `randomValue` to `conf`